app/scripts/controllers/newfromtemplate.js

```diff
@@ -143,7 +143,7 @@ angular.module('openshiftConsole')
               };
 
               if (!_.isEmpty(config.parameters)) {
```

Comment:
dont rely no the existence of config.parameters in order to set config.message, a template author may have a message but no parameters